daprdocs/content/en/developing-applications/integrations/open-service-mesh.md

```diff
@@ -0,0 +1,27 @@
+---
+type: docs
+title: "Running Dapr and Open Service Mesh together"
+linkTitle: "Open Service Mesh"
+weight: 4000
+description: "Learn how to run both Open Service Mesh and Dapr on the same Kubernetes cluster"
+---
+
+## Overview
+
+[Open Service Mesh (OSM)](https://openservicemesh.io/) is a lightweight, extensible, Cloud Native service mesh that allows users to uniformly manage, secure, and get out-of-the-box observability features for highly dynamic microservice environments.
```

Comment:
I would have this in lower case "cloud native"

Reply:
Fixed